client/src/containers/App/PublicContainer.js

```diff
@@ -46,6 +48,24 @@ class PublicContainer extends Component<Props> {
             )}
             path="/organisations/:organisation_id"
           />
+          <Route
+            exact
+            path="/events/:event_id/attendee/:attendee_id"
```

Comment:
@taleymirza @JavaEria this is the general user profile, we dont need the event/id in the url